docs/.vuepress/_snippets/install_os.md

```diff
@@ -12,7 +12,7 @@ Finally, you can follow the [Quickstart](#quickstart) to take it from here and c
 Run the following script to automatically detect the operating system and download Kuma:
 
 ```sh
-curl -L https://kuma.io/installer.sh | bash -
+curl -L https://kuma.io/installer.sh | VERSION={{ $page.latestVersion }} bash -
```

Comment:
Some stuff was fixed in https://github.com/kumahq/kuma-website/pull/947/files#diff-c1914417787e08bf72877184e8a52b9a713159e16493ddd6e483a6eabdb07c8dR8 - are there any other places that do this?

Reply:
No this was the remaining one.

Reply:
Ok I'm looking into them